index.js

```diff
@@ -13,6 +13,8 @@ class CoverageInstrumenter {
   async startInstrumenting() {
     this.session.connect();
 
+    await this.postSession('Debugger.enable');
```

Comment:
that should be disabled on exit then I assume?

It seems odd to me we need to enable the debugger, tho. any immediate ideas on why?